packages/block-library/src/post-title/edit.js

```diff
@@ -1,3 +1,24 @@
-export default function PostTitleEdit() {
-	return <h2>{ 'Hello world!' }</h2>;
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+export default function PostTitleEdit( { context } ) {
+	const { postType, postId } = context;
+
+	const post = useSelect(
+		( select ) =>
+			select( 'core' ).getEditedEntityRecord(
+				'postType',
+				postType,
+				postId
+			),
+		[ 'postType', 'postId' ]
```

Comment:
I'm pretty sure these shouldn't be strings.

Reply:
> I'm pretty sure these shouldn't be strings.

Good catch! Not sure how that happened. Fixed in 603c015d7.